src/main/resources/crafter/studio/database/upgrade/4.2.x/4.2.0.15-to-4.2.0.16.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2007-2024 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ALTER TABLE `audit`
+    ADD INDEX `audit_primary_target_value_idx` (`primary_target_value`) ;
```

Comment:
@phuongnq have we tested the impact of this index in the insertion performance?

Reply:
@jmendeza For a site with `470413` record in `audit` table, it took around 2.1s for a write API round-trip to the client (which includes the insert to audit log)
May need the QA team to double-test the impact on a big site with a larger audit table cc @sumerjabri

Reply:
A direct INSERT into `audit` with the same table size of `470413` is instant (~0.007s)
![Screenshot from 2024-11-08 14-11-43](https://github.com/user-attachments/assets/e6a1ed5f-57c9-4541-a35f-c405087c96e9)